examples/Training/python/Create_Image.py

```diff
@@ -15,13 +15,17 @@
 from Parse_OMERO_Properties import USERNAME, PASSWORD, HOST, PORT
 from Parse_OMERO_Properties import imageId
 
-# Create a connection
+"""
+start-code
+"""
+
+# Create a connection:
 # =================================================================
```

Comment:
Is there any reason why in the Python examples these "title comments" have almost all had a colon added? And is there a reason why the "underlining" can't be truncated to the length of the text above it? In the Java and MATLAB examples these comments look a lot neater.